packages/plugin-vue/src/index.ts

```diff
@@ -65,6 +65,20 @@ export interface Options {
    */
   reactivityTransform?: boolean | string | RegExp | (string | RegExp)[]
```

Comment:
We can deprecate `reactivityTransform` as well.

Reply:
Thanks! Updated.